ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/WithdrawalsValidator.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Hyperledger Besu Contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.hyperledger.besu.ethereum.api.jsonrpc.internal.methods.engine;
+
+import org.hyperledger.besu.ethereum.core.Withdrawal;
+import org.hyperledger.besu.ethereum.mainnet.TimestampSchedule;
+
+import java.util.List;
+import java.util.Optional;
+
+public class WithdrawalsValidator {
```

Comment:
The name of this class doesn't seem right. Looks like this is just calling the actual WithdrawalsValidator. Maybe it's a provider?

Reply:
haha yeh it's weird...kind of an API level version of the validator that handles getting the validator and then calling it

Reply:
I can try WithdrawalsValidatorProvider and then call validateWithdrawals in the api methods, which is probably what you meant :)